openvisualizer/moteProbe/moteProbe.py

```diff
@@ -131,6 +192,9 @@ def __init__(self,serialport=None,emulatedMote=None,iotlabmote=None):
         elif self.mode==self.MODE_IOTLAB:
             self.iotlabmote       = iotlabmote
             self.portname         = 'IoT-LAB{0}'.format(iotlabmote)
+        elif self.mode==self.MODE_TESTBED:
+            self.testbedmote      = testbedmote
+            self.portname         = 'opentestbed{0}'.format(testbedmote)
```

Comment:
rename `testbedmote` to `testbedmote_eui64`

Reply:
`self.portname         = 'opentestbed{0}'.format(testbedmote)`
to
self.portname         = 'opentestbed_{0}'.format(testbedmote)